testing/mysql/integration_test.go

```diff
@@ -92,10 +94,7 @@ func dropDatabase(t *testing.T, dbPath string) {
 	}
 }
 
-func prepareIntegrationTest(t *testing.T) string {
-	if databaseAdmin == nil {
-		t.Skip("Integration tests skipped")
-	}
+func prepareTmpDir(t *testing.T) string {
```

Comment:
Sure Ill fix this, I changed it because currently all it was doing was creating a tmp dir. But yes, for consistency we should revert to the original name.